content/news/030/index.md

```diff
@@ -70,6 +70,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Engine Updates
 
+### [Rusty Engine 4.0]
+
+![An example Rusty Engine game](rusty_engine4.png)
+_The updated collider visualization (white outlines) in Rusty Engine 4.0_
+
+[Rusty Engine] by [Nathan Stocks] is a game engine built on top of Bevy
+for people who are learning Rust.
+
+Notable new features in Version 4.0 include: no need for an `init!` macro, new
+collider visualization, text can now be rotated and scaled, and an updated
+[online tutorial]. See [the changelog for 4.0] for the full details. On the
+back end Bevy has been updated to 0.6 and we switched from
```

Comment:
> we

could you switch to a third-person perspective, please?

Reply:
✅ Done!